File: exs/mundo_1/ruby/014.rb
=begin
Desafio 014

Problema: Escreva um programa que converta uma temperatura
          digitando em graus Celsius e converta para graus Fahrenheit.

Resolução do problema:
=end

print"Qual a temperatura em Celcius? "
celcius = gets.chomp.to_f

fahrenheit = 1.8 * celcius + 32

puts"#{celcius} Celcius em Fahrenheit é #{fahrenheit}."

File: exs/mundo_1/javascript/014.js
/*
Desafio 014

Problema: Conversor de temperaturas: escreva um programa que converta uma temperatura 
          digitada em ºC para ºF

Resolução do problema:
*/

const input = require('readline-sync').question

let temperatura_C = Number(input('Informe a temperatura em °C: '))

let temperatura_F = 1.8 * temperatura_C + 32

console.log(`Temperatura: ${temperatura_F.toFixed(1)}°F`)
